include/deal.II/base/derivative_form.h

```diff
@@ -428,6 +428,30 @@ DerivativeForm<order, dim, spacedim, Number>::memory_consumption()
 
 
 
+/**
+ * Output operator for DerivativeForm. Print the elements consecutively, with a
+ * space in between, two spaces between rank 1 subtensors, three between rank 2
+ * and so on.
+ *
+ * @relatesalso DerivativeForm
+ */
+template <int order, int dim, int spacedim, typename Number>
+inline std::ostream &
+operator<<(std::ostream                                       &out,
+           const DerivativeForm<order, dim, spacedim, Number> &df)
```

Comment:
Is there somewhere a similar function that you have adopted here? Could you point me out to that one?